EIPS/eip-7777.md

```diff
@@ -0,0 +1,341 @@
+---
+eip: 7777
```

Comment:
```suggestion
eip: 7492
```
Assigning EIP number as PR number.  Please also rename the filename.

Reply:
Ah, so that's how I work out the eip number, thank you!